docs/key-issues/native-dependencies/blas_openmp.md

```diff
@@ -184,6 +184,39 @@ Linux package managers may do something similar
 or they may use a more a more specific way of managing BLAS and LAPACK, such as
 [Fedora using Flexiblas](https://docs.fedoraproject.org/en-US/packaging-guidelines/BLAS_LAPACK/#_user_level_selection).
 
+??? info "BLAS/LAPACK demuxing: FlexiBLAS, libblastrampoline & SciPy's `cython_blas`/`cython_lapack`"
+
+    Supporting multiple BLAS and LAPACK libraries is quite challenging,
+    because of API and ABI incompatibilities between them. In particular:
+
+    - API's using 32-bit (LP64) or 64-bit (ILP64) integers. Typically a
```

Comment:
Can clarify the integers are used as indices of the vectors/matrices, with maybe a link to the Wikipedia article https://en.wikipedia.org/wiki/64-bit_computing#64-bit_data_models for reference, since the acronyms are rather obscure for non-experts (especially LP64 for 32-bit integers 😅)

Reply:
> especially LP64 for 32-bit integers sweat_smile

yes, that one is 🤦🏼